Blockzilla/BrowserViewController.swift

```diff
@@ -470,6 +479,25 @@ class BrowserViewController: UIViewController {
             UIKeyCommand(input: "]", modifierFlags: .command, action: #selector(BrowserViewController.goForward), discoverabilityTitle: UIConstants.strings.browserForward),
         ]
     }
+    
+    private func shouldShowTrackerStatsShareButton() -> Bool {
+        if getNumberOfLifetimeTrackersBlocked() < 100 ||
+            NSLocale.current.identifier != "en_US" {
+            return false
+        }
+        
+        // 10% chance we show the button
+        // arc4random_uniform(10 returns an integer 0 through 9 (inclusive)
+        return arc4random_uniform(10) == 0
```

Comment:
So the way I understand this, you are a flipping a 10% coin every time a user presses Erase, so that means that all users would eventually see the message? We need to only show it to maximum of 10% of users - so we have to put users in buckets and save that bucket across the install. This is something we do for onboarding too.